src/mediarithmics/plugins/custom-action/CustomActionInterface.ts

```diff
@@ -0,0 +1,22 @@
+import { TimeStamp } from './../../api/reference/UserIdentifierInterface';
```

Comment:
Not a fan of the import path -> should we use a declaration from a "common" class?